vignettes/estimate_infections_workflow.Rmd

```diff
@@ -0,0 +1,259 @@
+Some of these can be mitigated using the routines available in _EpiNow2_ as described below, but others will cause biases in the results and need to be carefully considered when interpreting the results.
+
+# Set up
+
+We first load the _EpiNow2_ package.
+
+
+```r
+library("EpiNow2")
+```
+
+We then set the number of cores to use. We will want to run 4 MCMC chains in parallel so we set this to 4. If we had fewer than 4 available or wanted to run fewer than 4 chains (at the expense of some robustness), or had fewer than 4 computing cores available we could set it to that. To find out the number of cores available one can use the [detectCores](https://rdrr.io/r/parallel/detectCores.html) function from the `parallel` package.
+
+
+```r
+options(mc.cores = 4)
+```
+
+# Parameters
+
+Once a data set has been identified, a number of relevant parameters need to be considered before using _EpiNow2_.
+As these will affect any results, it is worth spending some time investigating what their values should be.
+
+## Delay distributions
+
+_EpiNow2_ works with different delays that apply to different parts of the infection and observation process.
+They are defined using a common interface with the `dist_spec()` function.
+For help with this function, see its manual page
+
+
+```r
+?dist_spec
+```
+
+In all cases, the distributions given can be *fixed* (i.e. have no uncertainty) or *variable* (i.e. have associated uncertainty).
+For example, to define a fixed gamma distribution with mean 3, standard deviation (sd) 1 and maximum value 10, you would write
+
+
+```r
+dist_spec(mean = 3, sd = 1, distribution = "gamma", max = 10)
+#> 
+#>   Fixed distribution with PMF [0.0038 0.15 0.39 0.3 0.12 0.03 0.006 0.00096 0.00013 1.6e-05]
+```
+
+If distributions are variable, the values with uncertainty are treated as [prior probability densities](https://en.wikipedia.org/wiki/Prior_probability) in the Bayesian inference framework used by _EpiNow2_, i.e. they are estimated as part of the inference.
+For example, to define a variable gamma distribution where uncertainty in the mean is given by a normal distribution with mean 3 and sd 2, and uncertainty in the standard deviation is given by a normal distribution with mean 1 and sd 0.1, with a maximum value 10, you would write
+
+
+```r
+dist_spec(
+  mean = 3, mean_sd = 2, sd = 1, sd_sd = 0.1, distribution = "gamma", max = 10
+)
+#> 
+#>   Uncertain gamma distribution with (untruncated) mean 3 (SD 2) and SD 1 (SD 0.1)
+```
+
+There are various ways the specific delay distributions mentioned below might be obtained.
+Often, they will come directly from the existing literature reviewed by the user and studies conducted elsewhere.
+Sometimes it might be possible to obatined them from existing databases, e.g. using the [epiparameter](https://github.com/epiverse-trace/epiparameter) R package.
```

Comment:
```suggestion
Sometimes it might be possible to obtain them from existing databases, e.g. using the [epiparameter](https://github.com/epiverse-trace/epiparameter) R package.
```